### PR TITLE
Workflow

### PR DESCRIPTION
**Pull request by Gy-Lu**
test CI

### CODE REVIEW

File: .github/workflows/release_bdist.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        run: |
+          pip install beautifulsoup4 requests packaging
+          python ./build_fastfold_wheel.py --torch_version $TORCH_VERSIONS
+        env:
+          TORCH_VERSIONS: ${{ inputs.torch_version }}
+      - name: 🚀 Deploy
+        uses: garygrossgarten/github-action-scp@release
+        with:
+          local: all_dist
+          remote: ${{ secrets.PRIVATE_PYPI_DIR }}
```
Comment (Gy-Lu) [inline .github/workflows/release_bdist.yml:77]: I think this line is wrong

File: README.md [hunk 2]
```diff
@@ -21,24 +21,50 @@ FastFold provides a **high-performance implementation of Evoformer** with the fo
...
+You can download FastFold with pre-built CUDA extensions.
 
 ```shell
-python setup.py install
+pip install fastfold -f https://release.colossalai.org/fastfold
+```
+
+## Use Docker
+### Pull from DockerHub
+You can directly pull the docker image from our [DockerHub page](https://hub.docker.com/r/hpcaitech/colossalai). The image is automatically uploaded upon release.
```
Comment (FrankLeeeee) [inline README.md:53]: This is not correct for alphafold.

Comment (Gy-Lu) [reply README.md:53]: right, I'll fix it.

File: .github/workflows/release_bdist.yml (hunk 1 above)
Comment (FrankLeeeee) [reply .github/workflows/release_bdist.yml:77]: I have added the secrets to this repository, it should work fine now.

Comment (Gy-Lu) [reply .github/workflows/release_bdist.yml:77]: ok thanks

**Review by Shenggan — APPROVED** [review]
LGTM

**Comment by Gy-Lu** [comment]
> LGTM

OK, I'll test it tonight.

**Pull request merged**